### PR TITLE
distribute initial

### PR DESCRIPTION
**Pull request by mikluke**

### CODE REVIEW

File: scripts/distribute-initial-from-db/main.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+			logrus.WithError(err).Errorf("failed to parse: %s", v)
+		}
+
+		msgs = append(msgs, bank.NewMsgSend(bt.From(), to, sdk.Coins{sdk.Coin{
+			Denom:  app.DefaultBondDenom,
+			Amount: sdk.NewInt(opts.InitialMainStakes),
+		}}))
+	}
+
+	logrus.Info("wait 30 seconds before broadcasting")
```
Comment (megaherz) [inline scripts/distribute-initial-from-db/main.go:79]: What for? To cross fingers? 🤞🏻

File: scripts/distribute-initial-from-db/main.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	"github.com/jessevdk/go-flags"
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+)
+
+var opts = struct {
+	Postgres string `long:"postgres" env:"POSTGRES" default:"host=localhost port=5432 user=postgres password=root sslmode=disable" description:"postgres dsn"`
+
+	BlockchainMainNode               string `long:"blockchain.main.node" env:"BLOCKCHAIN_MAIN_NODE" default:"http://zeus.testnet.decentr.xyz:26657" description:"decentr node address"`
```
Comment (megaherz) [inline scripts/distribute-initial-from-db/main.go:24]: mainnet instead of testnet everywhere

File: scripts/distribute-initial-from-db/main.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+	BlockchainMainChainID            string `long:"blockchain.main.chain_id" env:"BLOCKCHAIN_MAIN_CHAIN_ID" default:"testnet" description:"decentr chain id"`
+	BlockchainMainClientHome         string `long:"blockchain.main.client_home" env:"BLOCKCHAIN_MAIN_CLIENT_HOME" default:"~/.decentrcli" description:"decentrcli home directory"`
+	BlockchainMainKeyringBackend     string `long:"blockchain.main.keyring_backend" env:"BLOCKCHAIN_MAIN_KEYRING_BACKEND" default:"test" description:"decentrcli keyring backend"`
+	BlockchainMainKeyringPromptInput string `long:"blockchain.main.keyring_prompt_input" env:"BLOCKCHAIN_MAIN_KEYRING_PROMPT_INPUT" description:"decentrcli keyring prompt input"`
+	BlockchainMainGas                uint64 `long:"blockchain.main.gas" env:"BLOCKCHAIN_MAIN_GAS" default:"10" description:"gas amount"`
+	BlockchainMainFee                string `long:"blockchain.main.fee" env:"BLOCKCHAIN_MAIN_FEE" default:"1udec" description:"transaction fee"`
+	InitialMainStakes                int64  `long:"blockchain.main.initial_stakes" env:"BLOCKCHAIN_MAIN_INITIAL_STAKES" default:"1000000" description:"stakes count to be sent"`
```
Comment (megaherz) [inline scripts/distribute-initial-from-db/main.go:33]: It has to be 10DEC = 10000000

File: scripts/distribute-initial-from-db/main.go [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+	"github.com/sirupsen/logrus"
+)
+
+var opts = struct {
+	Postgres string `long:"postgres" env:"POSTGRES" default:"host=localhost port=5432 user=postgres password=root sslmode=disable" description:"postgres dsn"`
+
+	BlockchainMainNode               string `long:"blockchain.main.node" env:"BLOCKCHAIN_MAIN_NODE" default:"http://zeus.testnet.decentr.xyz:26657" description:"decentr node address"`
+	BlockchainMainFrom               string `long:"blockchain.main.from" env:"BLOCKCHAIN_MAIN_FROM" description:"decentr account name to send stakes" required:"true"`
+	BlockchainMainTxMemo             string `long:"blockchain.main.tx_memo" env:"BLOCKCHAIN_MAIN_TX_MEMO" description:"decentr tx's memo'"`
+	BlockchainMainChainID            string `long:"blockchain.main.chain_id" env:"BLOCKCHAIN_MAIN_CHAIN_ID" default:"testnet" description:"decentr chain id"`
```
Comment (megaherz) [inline scripts/distribute-initial-from-db/main.go:27]: mainnet-1

File: scripts/distribute-initial-from-db/main.go (hunk 1 above)
Comment (mikluke) [reply scripts/distribute-initial-from-db/main.go:79]: yep, to interrupt broadcasting

**Review by megaherz — APPROVED** [review]

**Pull request merged**